### PR TITLE
#194 skip constraint rule

### PR DESCRIPTION
**Pull request by dmzaytsev**
for #194 

* created a test  to reproduce the issue
* skipped the constraint  "HV000151: A method overriding another method must not alter the parameter constraint configuration" by catching the exception.
* created a puzzle to find a better option to disable the constraint

### CODE REVIEW

**Comment by dmarkov** [comment]
@dmzaytsev Thanks, someone will review your pull request soon

**Comment by dmarkov** [comment]
@krzyk it's yours, please review

File: src/main/java/com/jcabi/aspects/aj/MethodValidator.java [hunk 1]
```diff
@@ -148,18 +149,28 @@ public void after(final JoinPoint point, final Object result) {
      * @param object Object at pointcut
      * @param method Method at pointcut
      * @param args Parameters of the method
+     * @todo We need to find a way to disable the constraint rule
```
Comment (krzyk) [inline src/main/java/com/jcabi/aspects/aj/MethodValidator.java]: @dmzaytsev todo has incorrect format, it should contain issue id and time estimate

**Comment by krzyk** [comment]
@dmzaytsev just one issue with todo formatting

**Comment by dmzaytsev** [comment]
@krzyk done
please take a look

**Comment by krzyk** [comment]
@dmzaytsev you forgot about the hash in todo :)

**Comment by dmzaytsev** [comment]
@krzyk indeed :)
sorry

**Comment by dmzaytsev** [comment]
@krzyk done

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by dmarkov** [comment]
@krzyk Many thanks! **19 mins** were added to your account in Transaction ID `74441266` (task took 71 hours and 27 mins)

extra minutes for review comments (c=4)

+19 to your rating, your total score is [+3392](http://www.netbout.com/b/35035?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy

**Pull request merged**